.gitlab-ci.yml

```diff
@@ -34,15 +37,50 @@ SC_Client::Unity::build:
     - unity
     - win10
 
+SC_Client::Web::build:
+  stage: build
+  when: always
+  script:
+    # Update Version.txt with latest information before packaging
+    - python ./Scripts/UpdateVersionFile.py --path ${env:SC_CFW_PATH}/Version.txt --swVer ${env:SC_CFW_VERSION} --apiVer ${env:SC_API_VERSION} --ref ${env:CI_COMMIT_SHA}
+    # Build Client
+    - cd ${env:SC_CFW_PATH}
+    - npm i
+    - ./node_modules/.bin/gulp build
+    # Package Client
+    - cd ${env:CI_PROJECT_DIR}
+    - mkdir ./Client_Release
+    - mkdir ./Client_Release/dist
+    - cp ${env:SC_CFW_PATH}/Version.txt ./Client_Release
+    - cp -r ${env:SC_CFW_PATH}/build/dist ./Client_Release/
+  artifacts:
+    name: "Screen_Control_Client_for_Web_${env:SC_CFU_VERSION}_${env:CI_COMMIT_SHA}"
+    paths:
+      - ./Client_Release/*
+    expire_in: 2 weeks
+    when: always
+  tags:
+    - unity
+    - win10
```

Comment:
This refers to the web client, id that correct?

Reply:
These tags indicate which build machines the job can work on, and the Unity runners already have Node installed on them so this works fine 👍

Reply:
Sorry I should have been more clear, this branch/work is related to the Service, why is there a large chunk of web-specific changes?

Reply:
Ohhhhhh. Because I extended the Web branch to do this and the web branch hasn't yet been merged 😅